### ISSUE TITLE
Is the getting started example "getting started" enough?

### ISSUE THREAD

**Issue by MatteoRobbiati** [qibogang/qibogang.github.io#64]
Maybe a deeper check of the getting started tutorial can be useful.

**Comment by AleCandido** [comment]
Here the help of someone with more experience on Qibo would be helpful @andrea-pasquale

**Comment by andrea-pasquale** [comment]
Sure @AleCandido @MatteoRobbiati I will have a look. I've already written two different tutorials on Qibo with the help of @stavros11 you can find them in [this repository](https://github.com/andrea-pasquale/qibo_tutorial) if you want to have a look.

**Comment by AleCandido** [comment]
Thank you @andrea-pasquale!
There is no hurry, but the final goal is to eventually move all the resources to https://github.com/qiboteam/tutorials/

**Comment by andrea-pasquale** [comment]
Yes, I know :) . This was just a directory that I created in order to store the tutorials that I did for two talks.
